living_documentation_regime/action_inputs.py

```diff
@@ -164,4 +172,4 @@ def validate_repositories_configuration(self) -> None:
                 )
                 sys.exit(1)
 
-        logger.debug("Repositories configuration validation successfully completed.")
+        logger.debug("User configuration validation successfully completed.")
```

Comment:
Think about adding debug log lines printing used configuration into log.
In debug regime you can clearly see what values were used for run.

Reply:
User configuration added in [7a9fd1a](https://github.com/AbsaOSS/living-doc-generator/pull/83/commits/7a9fd1af47281449f919303055ec58f7b99722d8).